### DISCUSSION TITLE
Proprietary license

### DISCUSSION THREAD

**Discussion by Fireshtorm1k** [zeroc-ice/ice#2743] [Q&A] [answered]
Can you provide a price list for the Zeroc Ice license. Or where I can get acquainted with it. I filled out the form here https://zeroc.com/contact , but I did not receive a response

**Comment by externl** [comment, marked as answer]
Hi @Fireshtorm1k, 

Please contact us at [info@zeroc.com](mailto:info@zeroc.com). 

We received your contact forum message, however our email server was unable to deliver a response to the provided email address.
